README.md

```diff
@@ -53,7 +54,7 @@ of parameters down to the valid set.
 For example,
 
 ```r
-run_benchmark(write_file, source = "nyctaxi_2010-01")
+arrowbench::run_benchmark(write_file, source = "nyctaxi_2010-01")
```

Comment:
Looks like `library(arrowbench)` needs to be added to these instructions.

Rather than English instructions, perhaps we should aim for complete instructions that you can copy/paste from without assuming anything.

Like:

```
$ R
> install.packages('remotes')
> remotes::install_github("ursa-labs/arrowbench")
> library(arrowbench)
> run_benchmark(write_file, source = "nyctaxi_2010-01")
```

Although, I gather from slack that the above might need to be modified since it assumes arrow is already installed?

And then also a contributors example that does something like the following.

```
$ git clone https://github.com/ursa-labs/arrowbench.git
$ cd arrowbench
$ R
> install.packages('remotes')
> remotes::install_deps(".")
> library(arrowbench)
> run_benchmark(write_file, source = "nyctaxi_2010-01")
```

Grain of salt: I don't know R, just trying to following the README. Thanks for your patience!

My current error:

```
Running 28 benchmarks with 1 iterations:
            source  format  compression       input cpu_count
1  nyctaxi_2010-01 parquet uncompressed arrow_table         1
2  nyctaxi_2010-01 feather uncompressed arrow_table         1
3  nyctaxi_2010-01     fst uncompressed arrow_table         1
4  nyctaxi_2010-01 parquet       snappy arrow_table         1
7  nyctaxi_2010-01 parquet         zstd arrow_table         1
8  nyctaxi_2010-01 feather         zstd arrow_table         1
9  nyctaxi_2010-01     fst         zstd arrow_table         1
10 nyctaxi_2010-01 parquet uncompressed  data_frame         1
11 nyctaxi_2010-01 feather uncompressed  data_frame         1
12 nyctaxi_2010-01     fst uncompressed  data_frame         1
13 nyctaxi_2010-01 parquet       snappy  data_frame         1
16 nyctaxi_2010-01 parquet         zstd  data_frame         1
17 nyctaxi_2010-01 feather         zstd  data_frame         1
18 nyctaxi_2010-01     fst         zstd  data_frame         1
19 nyctaxi_2010-01 parquet uncompressed arrow_table         4
20 nyctaxi_2010-01 feather uncompressed arrow_table         4
21 nyctaxi_2010-01     fst uncompressed arrow_table         4
22 nyctaxi_2010-01 parquet       snappy arrow_table         4
25 nyctaxi_2010-01 parquet         zstd arrow_table         4
26 nyctaxi_2010-01 feather         zstd arrow_table         4
27 nyctaxi_2010-01     fst         zstd arrow_table         4
28 nyctaxi_2010-01 parquet uncompressed  data_frame         4
29 nyctaxi_2010-01 feather uncompressed  data_frame         4
30 nyctaxi_2010-01     fst uncompressed  data_frame         4
31 nyctaxi_2010-01 parquet       snappy  data_frame         4
34 nyctaxi_2010-01 parquet         zstd  data_frame         4
35 nyctaxi_2010-01 feather         zstd  data_frame         4
36 nyctaxi_2010-01     fst         zstd  data_frame         4
Running source=nyctaxi_2010-01 format=parquet compression=uncompressed input=arrow_table cpu_count=1 name=write_file
Error: lexical error: invalid char in json text.
                                       Error in force(x) : object 'out
                     (right here) ------^
```

Reply:
Upgrading from from R version 3.4.4 to R version 4.0.3, fixed the above error, so perhaps we need to document a minimum R version.

Reply:
R 3.5 is where the `deparse` options were changed in this way, so we can set that as the min in the DESCRIPTION. Package will fail with a useful message if you try to install on an older version than is allowed there, so I think that is sufficient documentation.